lib/xeroizer/generic_application.rb

```diff
@@ -58,6 +59,8 @@ def initialize(consumer_key, consumer_secret, options = {})
         @rate_limit_sleep = options[:rate_limit_sleep] || false
         @rate_limit_max_attempts = options[:rate_limit_max_attempts] || 5
         @default_headers = options[:default_headers] || {}
+        @before_request = options.delete(:before_request)
+        @after_request = options.delete(:after_request)
```

Comment:
Note that these have to be deleted from `options`, rather than just referenced like the others above, because otherwise they get passed straight into the OAuth::Consumer object and end up causing problems. It's kind of worrying, in general, that the Xeroizer options all get passed straight to OAuth::Consumer; shouldn't all of these Xeroizer-specific options be `delete`d here?